File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.java
class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        int index = 0;
        String str[] = new String[intStrs.length];
        for(int i = 0; i < intStrs.length; i++){
            char tempch[] = new char[l];
            for(int j = 0; j < l; j++){
                tempch[j] = intStrs[i].charAt(s+j);
            }
            str[index++] = new String(tempch);
        }
        int count = 0;
        int trueCount = 0;
        for(int i = 0; i < str.length; i++){
            if(Integer.valueOf(str[i]) > k){
                trueCount++;
            }
        }
        int answer[] = new int[trueCount];
        for (int i = 0; i < str.length; i++) {
            if (Integer.valueOf(str[i]) > k) {
                answer[count++] = Integer.valueOf(str[i]);
            }
        }

        
        return answer;
    }
}

File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.cs
using System;

public class Solution {
    public int[] solution(string[] intStrs, int k, int s, int l) {
        int index = 0;
        string[] str = new string[intStrs.Length];
        for(int i = 0; i < intStrs.Length; i++){
            char []tempch = new char[l];
            for(int j = 0; j < l; j++){
                tempch[j] = intStrs[i][s+j];
            }
            str[index++] = new String(tempch);
        }
        int count = 0;
        int trueCount = 0;
        for(int i = 0; i < str.Length; i++){
            if(Convert.ToInt32(str[i]) > k){
                trueCount++;
            }
        }
        int[] answer = new int[trueCount];
        for (int i = 0; i < str.Length; i++) {
            if (Convert.ToInt32(str[i]) > k) {
                answer[count++] = Convert.ToInt32(str[i]);
            }
        }

        
        return answer;
    }
}